test/gtest/ucp/test_ucp_am.cc

```diff
@@ -328,12 +317,12 @@ class test_ucp_am_nbx : public test_ucp_am_base {
 
     ucp_datatype_t make_dt(int dt)
     {
-        if (dt & DT_CONTIG) {
+        if (dt == UCP_DATATYPE_CONTIG) {
            return ucp_dt_make_contig(1);
-        } else if (dt & DT_IOV) {
+        } else if (dt == UCP_DATATYPE_IOV) {
            return ucp_dt_make_iov();
         } else {
-            EXPECT_TRUE(dt & DT_GENERIC);
+            EXPECT_EQ(UCP_DATATYPE_GENERIC, dt);
```

Comment:
ASSERT_EQ?

Reply:
not possible to use gtest's ASSERT since it returns from func as void, will use ucs_assert()

Reply:
> not possible to use gtest's ASSERT since it returns from func as void, will use ucs_assert()

I see, thank you for the explanation!